src/lib/components/_utilities/locale-strings.story.svelte

```diff
@@ -1,8 +1,11 @@
 <script lang="ts">
   import Centered from '$components/primitives/box/centered.svelte';
+  import Text from '$components/primitives/text/text.svelte';
   import T from '$components/_utilities/locale-strings.svelte';
 </script>
 
 <Centered>
-  <T key="continueWith" />
+  <Text>
+    <T key="continueWith" />
+  </Text>
```

Comment:
This story and the one below just needed to be wrapped so it had the proper theme treatment in the rendered storybook component.